sound/soc/sof/core.c

```diff
@@ -636,11 +647,23 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 	mutex_init(&sdev->ipc_client_mutex);
 	mutex_init(&sdev->client_event_handler_mutex);
 
-	/* set default timeouts if none provided */
+	/* set IPC timeout */
```

Comment:
@ujfalusi there's a checkpatch warning related to the indentation and it looks legit. Do you mind checking? Perhaps moving the new parts after the following if/else to override the default value might be better?

Reply:
@ranj063, yes, probably that is better, I'll do that.